react/utils/withDevice.tsx

```diff
@@ -40,16 +40,18 @@ const useDevice = (hints: RenderHints) => {
     isMobile: hints.mobile,
   }
 
-  const clientDevice = {
-    type: isScreenLarge
-      ? Device.desktop
-      : isScreenMedium
-      ? Device.tablet
-      : Device.phone,
-    isMobile: !isScreenLarge,
-  }
+  return serverDevice
+
+  // const clientDevice = {
+  //   type: isScreenLarge
+  //     ? Device.desktop
+  //     : isScreenMedium
+  //     ? Device.tablet
+  //     : Device.phone,
+  //   isMobile: !isScreenLarge,
+  // }
 
-  return isSSR ? serverDevice : clientDevice
+  // return isSSR ? serverDevice : clientDevice
```

Comment:
fixes an issue caused by concurent mode, where the page blinks into mobile mode before turning back into desktop. have to look into it